products/opensuse/main.pm

```diff
@@ -654,19 +665,50 @@ sub load_extra_tests () {
     return 0;
 }
 
+sub load_otherDE_tests() {
+    if (get_var("DE_PATTERN")) {
+        my $de = get_var("DE_PATTERN");
+        loadtest "console/consoletest_setup.pm";
+        loadtest "console/hostname.pm";
+        loadtest "console/zypper_clear_repos.pm";
+        loadtest "console/install_otherDE_pattern.pm";
+        loadtest "console/consoletest_finish.pm";
+        loadtest "x11/${de}_reconfigure_openqa.pm";
+        loadtest "x11/reboot_icewm.pm";
+        # here comes the actual desktop specific test
+        if ($de =~ /^awesome$/)       { load_awesome_tests(); }
+        if ($de =~ /^enlightenment$/) { load_enlightenment_tests(); }
+        if ($de =~ /^mate$/)          { load_mate_tests(); }
+        if ($de =~ /^lxqt$/)          { load_lxqt_tests(); }
+        loadtest "x11/shutdown.pm";
+        return 1;
+    }
+    return 0;
+}
+
+sub load_awesome_tests() {
+    loadtest "x11/awesome_menu.pm";
+    loadtest "x11/awesome_xterm.pm";
+}
+
+sub load_enlightenment_tests() {
+    loadtest "x11/enlightenment_first_start.pm";
+    loadtest "x11/terminology.pm";
+}
+
+sub load_lxqt_tests() {
+}
```

Comment:
That's because I did not get feedback from the LXQt maintainer yet as to what he'd want tested in plus... can be extended at any moment